### PR TITLE
CFE-2740: Do not move obstructions in warn policy mode

### PR DESCRIPTION
**Pull request by amousset**
https://tracker.mender.io/browse/CFE-2740

### CODE REVIEW

**Comment by nickanderson** [comment]
Looks like the test passes https://travis-ci.org/cfengine/core/jobs/315826346#L3571

File: cf-agent/files_links.c [hunk 1]
```diff
@@ -397,6 +402,12 @@ PromiseResult VerifyHardLink(EvalContext *ctx, char *destination, const char *so
 
     Log(LOG_LEVEL_INFO, "'%s' does not appear to be a hard link to '%s'", destination, to);
 
+    if (!EnforcePromise(attr.transaction.action))
+    {
+        Log(LOG_LEVEL_WARNING, "Hark link '%s' -> '%s' should be created", destination, to);
```
Comment (jimis) [inline cf-agent/files_links.c]: typo "hark link"

**Comment by amousset** [comment]
done!

**Comment by nickanderson** [comment]
@jimis your changes have been done, squashed and rebased. Shall we merge?

**Pull request merged**